### PR TITLE
Update pulse.md

### PR DESCRIPTION
**Pull request by asibyl**
Adding more details on how to read event_count and event_dau lifts in Pulse.

### CODE REVIEW

**Comment by jasonwzm** [comment]
🚀 Deployed on https://630f9051176a30379adae7d2--cozy-fox-0defec.netlify.app

File: docs/metrics/pulse.md [hunk 1]
```diff
@@ -22,7 +22,46 @@ Statsig computes experiment results, also known as Statsig's **Pulse** results,
 | user: retention metrics |  Overall average retention rate   | Overall average retention rate  | Participating users |
 | user: L7, L14, L28 |  Not shown   | Average L-ness value per user per day  | All users |
 
+## Event Count and Event DAU in Pulse
+As independent metrics,
```
Comment (maggie-ss) [inline docs/metrics/pulse.md:26]: > As independent metrics, 

Not sure what this means

File: docs/metrics/pulse.md [hunk 2]
```diff
@@ -22,7 +22,46 @@ Statsig computes experiment results, also known as Statsig's **Pulse** results,
...
 To reduce the impact of outliers, Statsig caps *event_count* and *sum* metric types at the 99.9th percentile. This mitigates the risk of bots and extreme values significantly swaying experiment results.
 
+
+## Frequently Asked Questions
+
+**1. Is it possible for a ratio metric to move in the opposite direction than both the numerator and denominator metrics?** 
+
+Yes, it is possible for the ratio to rise while both the numerator and denominator metrics decline. For example, this happens when the denominator is falls more than the numerator. As a best practice, Statsig recommends tracking the numerator and denominator as independent metrics when monitoring ratio metric. Ratio metrics are often subject to statistical noise and can be tricky to use for obtaining a statistically significant result.   
+
+**2. For ratio metrics using event_dau, how does Statsig determine _participating users_?**
```
Comment (maggie-ss) [inline docs/metrics/pulse.md:64]:
```suggestion
**2. For ratio metrics, how does Statsig determine _participating users_?**
```

File: docs/metrics/pulse.md [hunk 3]
```diff
@@ -22,7 +22,46 @@ Statsig computes experiment results, also known as Statsig's **Pulse** results,
...
+
+Ratio metrics that use event_dau include user-days only when the denominator is non-zero i.e. the user must have triggered the denominator event on a given day to be included in the daily ratio.
```
Comment (maggie-ss) [inline docs/metrics/pulse.md:66]:
```suggestion
Ratio metrics are computed only for users that have a non-zero value in the denominator, i.e. the user must have triggered the denominator event on a given day to be included in the daily ratio.   Users that don't trigger the denominator event during an experiment are not included in the test vs. control comparison of a ratio metric.
```

**Review by maggie-ss — APPROVED** [review]

**Pull request merged**